tools/shell-commands/src/main/resources/requestBody/2.0.0/event_migrate.painless

```diff
@@ -18,7 +18,7 @@
 /* Handle login events */
 if ('login' == ctx._source.eventType) {
     /* Look for empty scope */
-    if (ctx._source.scope == '') {
+    if (ctx._source.scope == '' || ctx._source.scope == null) {
```

Comment:
It would work for user who are still in a using Unomi < 2.0.0 but for users who are already in version 2.0.0, this migration script will not be executed.
It would be better to have another script which will be executed in 2.5.0.

Reply:
Indeed, it has been moved to a 2.5.0 migration step